puppeteer/basic/tests/login.js

```diff
@@ -0,0 +1,58 @@
+const { assert } = require('chai')
+const puppeteer = require('puppeteer')
+const {
+  wrapPlaywright,
+  PuppeteerController,
+  puppeteerConfig
+} = require('@axe-core/watcher')
+
+/* Get your configuration from environment variables. */
+const { API_KEY, SERVER_URL = 'https://axe.deque.com' } = process.env
```

Comment:
NBD - We have an `assert` for the typescript example, we should add it here or remove it 
```suggestion
/* Get your configuration from environment variables. */
const { API_KEY, SERVER_URL = 'https://axe.deque.com' } = process.env
assert(API_KEY, 'API_KEY is required')
```